docs/markdownlint-rules.yaml

```diff
@@ -0,0 +1,3 @@
+line-length:
+  line_length: 120
+  tables: false
```

Comment:
Maybe we should disable rule 40 https://github.com/DavidAnson/markdownlint/blob/main/doc/md040.md

GitHub is smart enough to detect the language

Reply:
nvm, this will go into our website docs. Better be safe than sorry ;)